Automattic/wp-calypso
# Jetpack: remove Personal plan from plans table

### Changes proposed in this Pull Request

This PR removes the Jetpack Personal plan from the plans table, unless the customer already owns it. The table appears in 2 places: in the _Plans_ section and Jetpack connect.

### Testing instructions

1. Download the PR
2. Select a self-hosted Jetpack site with a Free plan
3. Go to the _Plans_ section (`/plans/:site/`), check that the Personal plan has been removed, and that the Premium plan contains the "Daily Backup",  "Spam Protection" and "All Free features" features (for both monthly and yearly plans)
4. Go to `/jetpack/connect/plans/yearly/:site/` and do the same check
5. Repeat with a site that has the Personal plan. Check that the Personal plan is present in the pricing table and that the Premium features have not been updated.

### Screenshots

### Plans
Site with Personal plan
<img width="672" alt="Screen Shot 2020-07-28 at 4 07 15 PM" src="https://user-images.githubusercontent.com/1620183/88716728-7eb77400-d0ed-11ea-9407-b92188dcff6a.png">

Site with other plan
<img width="674" alt="Screen Shot 2020-07-28 at 4 06 39 PM" src="https://user-images.githubusercontent.com/1620183/88716745-8545eb80-d0ed-11ea-8b0d-2a8a173b5009.png">

### Jetpack connect
<img width="871" alt="Screen Shot 2020-07-21 at 12 23 18 PM" src="https://user-images.githubusercontent.com/1620183/88080957-c66a5880-cb4d-11ea-9e09-7f071f7ce744.png">
